src/sass/button.scss

```diff
@@ -1,5 +1,4 @@
 @mixin link-button {
-    text-decoration: none;
     text-align: center;
     line-height: 100%;
     text-transform: capitalize;
```

Comment:
should we still keep the text capitalization?

Reply:
That would be a different project. There is already a card for it.